src/validate_pyproject/formats.py

```diff
@@ -378,7 +378,31 @@ def int(value: builtins.int) -> bool:
     return -(2**63) <= value < 2**63
 
 
-def SPDX(value: str) -> bool:
-    """Should validate eventually"""
-    # TODO: validate conditional to the presence of (the right version) of packaging
-    return True
+try:
+    try:
+        from packaging import licenses as _licenses
+    except ImportError:  # pragma: no cover
+        # let's try setuptools vendored version
+        from setuptools._vendor.packaging import (  # type: ignore[no-redef]
+            licenses as _licenses,
+        )
```

Comment:
I don't think we need to cover this case...

- Newer versions of `setuptools` use this strategy for the `_vendor` directoru:

  ```python
  sys.path.extend(((vendor_path := os.path.join(os.path.dirname(os.path.dirname(__file__)), 'setuptools', '_vendor')) not in sys.path) * [vendor_path])  # fmt: skip
  ```

  So `packaging` will be available directly, without the need for prefixing it with `setuptools._vendor`.

- Older versions of setuptools will contain older versions of `packaging` so importing `setuptools._vendor.packaging.licenses` will always fail, right?